src/main/java/ua/softserveinc/tc/constants/UserConstants.java

```diff
@@ -2,6 +2,8 @@
 
 
 public final class UserConstants {
+  public static final String USER_CALENDAR = "user-calendar";
+
```

Comment:
this constant is common to
public static final String MANAGER_CALENDAR = "manager-calendar";